getting-started.md

```diff
@@ -0,0 +1,61 @@
+
+Note that the OCS example project is only meant for testing, and that if you want to restart the stack, the entire stack should be removed and recreated.
+
+To stop the example project and remove the Docker containers that were created:
+
+```bash
+docker-compose down
+```
+
+What can you do with the OCS example stack?
+- Submit observation requests using either the Observation Portal API or the web frontend
+- Watch as the Adaptive Scheduler turns observation requests into scheduled observations
+- View all scheduled observations and observation requests, using either the Observation Portal API or the web frontend
+- Update observatory resources in ConfigDB to specify resources available for scheduling and watch as the scheduler shuffles the observing schedule to only place observations on schedulable resources
+- Submit periods of scheduled downtime to DowntimeDB and watch as the scheduler shuffles around the observing schedule, not placing any observation requests during periods of scheduled downtime
+
+Once you've gotten a feel for the OCS, here are some ways you can expand the capability of this stack, or customize it for your own needs:
+- Add ElasticSearch to your stack in which you can store data that the Adaptive Scheduler will use to keep track of whether a telescope is available for scheduling or not, for reasons such as bad weather or instrument failure.
+- Ultimately you can test out integrating the OCS into your system by simulating updating observation statuses using the Observation Portal API.
+
+## Next steps
+
+To learn more about OCS capabilities and how to use it, you can take a look through the [OCS Observation Portal]({{ site.baseurl }}{% link components/index.md %}) observing request language documentation to understand how you can use the language to describe your observatory's observing capabilities. You can also take a look through how data is modelled in the [OCS ConfigDB]({{ site.baseurl }}{% link components/index.md %}) to understand how you can model the resources in your own observatory. Beyond that, browsing the [Components]({{ site.baseurl }}{% link components/index.md %}) section will give you a better idea of the purpose of all the components of the OCS and how they work together, taking a look through different [Topics of Interest]({{ site.baseurl }}{% link topics/index.md %}) sections might help answer some more specific questions, and taking a look through the [API documentation]({{ site.baseurl }}{% link api/index.md %}) of the different projects will show you how to use the OCS APIs.
```

Comment:
"you can also take a look through how data is modeled..." -> "you can also take a look at the observatory configuration..."